simapp/app.yaml

```diff
@@ -46,3 +46,16 @@ modules:
   - name: params
     config:
       "@type": cosmos.params.module.v1.Module
+
+  - name: bank
+    config:
+      "@type": cosmos.bank.module.v1.Module
+#      uncomment to provide a block list different from the modules mentioned in auth.module_account_permissions
+#      blocked_module_accounts:
```

Comment:
above we have 
```
 module_account_permissions:
        - account: fee_collector
        - account: distribution
```
does this mean they can receive funds?

Reply:
The default (existing) behavior is to disallow funds transfer to any of the module accounts in `module_account_permissions` including those you mentioned.

Reply:
The block that is commented out here is new functionality, providing a few to override `bank` behavior to differ from `auth` configuration.  Previously they were both being injected from code in SimApp.

Reply:
My thinking was that it might be good to demonstrate the full functionality of `app.yaml` in comments, kind of like the [lein sample.project.clj](https://github.com/technomancy/leiningen/blob/master/sample.project.clj) but perhaps this is better as a documentation task, not the live `SimApp` config.

Reply:
Yeah, we actually want to discourage this type of customization IMHO. This particular design issue has caused bugs. Honestly I can't wait to redesign it but for now we just need to limit damage